### PR TITLE
Iteration2/rh

### PR DESCRIPTION
**Pull request by RonLHead**
Completed iteration 2.

Web page now opens to a login page where the user enters their name. Once the user clicks on the submit button, the page toggles to the main page with a welcome message that interpolates "Welcome" plus the input name.

### CODE REVIEW

**Comment by RonLHead** [comment]
Looks awesome! Great job!

File: main.js [hunk 1]
```diff
@@ -2,8 +2,13 @@ var receiveMessage = document.querySelector('.receive-button');
 var radioAffirmation = document.querySelector('#affirmation');
 var radioMantra = document.querySelector('#mantra');
 var meditationBell = document.querySelector('#meditate-bell');
-var displayMessage = document.querySelector('#message-display')
-// var message = "";
+var displayMessage = document.querySelector('#message-display');
+var inputName = document.querySelector('#input');
+var welcomePage = document.querySelector('.welcome-box');
```
Comment (alexmfritz) [inline main.js:7]: Stretch goal for intermission: querySelect all elements by id attributes

querySelector('#element')
getElementById('element')

File: main.js [hunk 2]
```diff
@@ -41,20 +46,31 @@ var mantras = [
...
 
+inputName.addEventListener('click', showPage);
 
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
 
 function getMessage(event) {
   event.preventDefault();
+
```
Comment (alexmfritz) [inline main.js:57]: Consider looking into different ways the getRandomIndex() function can be written to potentially clean up the readability -

affirmations[getRandomIndex(affirmations)] - might be a little confusing when used in denser functions

**Comment by alexmfritz** [comment]
As a whole - this is all super clean!!! Psyched to see what you can churn out towards the end of the inning. Most of these comments are things to consider, by no means necessary or needed now. But could be great habits to build sooner than later :)

**Pull request merged**